src/card/settings/custom/CardSettingsContentPropertySelect.tsx

```diff
@@ -234,6 +245,49 @@ const NeoCardSettingsContentPropertySelect = ({
             handleFreeTextNameSelectionUpdate(value);
           }}
         />
+      ) : settings.type == 'Custom Query' ? (
+        <>
+          <div>
+            <NeoField
+              label={'Name'}
+              key={'query'}
+              value={settings?.entityType || ''}
+              defaultValue={''}
+              placeholder={'Enter a parameter name here...'}
+              style={{ width: 335, marginLeft: '5px', marginTop: '0px' }}
+              onChange={(value) => {
+                setLabelInputText(value);
+                handleNodeLabelSelectionUpdate(value);
+                handleFreeTextNameSelectionUpdate(value);
+              }}
+            />
+            <NeoCodeEditorComponent
```

Comment:
@BennuFire 
Looks good! and seems to work if i specify a `display` in the output like so:
```
UNWIND range(1,5) as X
RETURN $input+X as value, $input+Y as display
```
However, in the simple scenario, without a display it doesn't seem to work. 
Could we make it so both work? Is there possibly something broken with the changes you made to compatibility mode?

Another issue, when I open/close the settings, the query seems to reset to `RETURN true` sometimes and delete the custom query. 

When I specify an invalid query (sporadically) the component breaks altogether, any idea what this could be?
![image](https://user-images.githubusercontent.com/1754968/231862620-f92795a2-bace-47a6-82f9-d2f92bd4e5f4.png)